scripts.js

```diff
@@ -0,0 +1,17 @@
+var text = "Papugi koszą trawę, unosząc się nad ziemią 15 centymetrów, na czerwonych dywanach.";
+console.log(text);
+
+var animal = "Zielone słonie";
+console.log(animal);
+
+var animalUpperCased = animal.toUpperCase();
+console.log(animalUpperCased);
+
+var textReplace = text.replace("Papugi", animalUpperCased);
+console.log(textReplace);
+
+console.log("dlugosc polowy zmiennej textReplace:", textReplace.length/2);
+
+var partOfTextReplace = textReplace.slice(0, 45);
```

Comment:
Twój program zadziała tylko dla tego konkretnego zdania. Jeśli `text` ulegnie zmienie, to już nie wyświetlisz idealnie połowy tekstu. Spróbuj zrobić to tak żeby tekst dzielić na pół bez względu na to jak długi on będzie.